test/menu_options_formatter_test.exs

```diff
@@ -0,0 +1,15 @@
+defmodule MenuOptionsFormatterTest do
+  use ExUnit.Case
+  doctest ToDo
+
+  test "to do menu options displayed with no items" do
+    assert ToDo.MenuOptionsFormatter.menu_options == [
+      "To Do List:",
+      [],
+      "[aA] Add New To-Do Items",
+      "[bB] List All Items",
+      "[cC<item_number>] Edit Item",
+      "[dD<item_number>] Delete Item"
+      ]
```

Comment:
There's duplication between the test code and the production code. How can we minimize this?